app/controllers/application/edit.js

```diff
@@ -26,7 +26,10 @@ export default class EditController extends Controller {
   }
 
   @action
-  submit() {
+  submit(event) {
+    if (event) {
```

Comment:
I don't think these are necessary

Reply:
Yeah I know I added them for a reason but I no longer remember why so I'll just remove them

Reply:
The docs talk about them but I don't think this relevant for us https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/#toc_the-on-modifier